components/ContinentScreen/ContinentScreen.utils.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Given an array of countries, return the number of unique languages
+ *
+ * @param {object[]} [countries]
+ * @returns {number}
```

Comment:
* @returns {number} - Number of unique languages

This helps code editors to give useful values when function is being used